src/rpc/server.cpp

```diff
@@ -467,14 +467,43 @@ std::string HelpExampleRpc(const std::string& methodname, const std::string& arg
         "\"method\": \"" + methodname + "\", \"params\": [" + args + "] }' -H 'content-type: text/plain;' http://127.0.0.1:8232/\n";
 }
 
-string experimentalDisabledHelpMsg(const string& rpc, const string& enableArg)
-{
-    return "\nWARNING: " + rpc + " is disabled.\n"
-        "To enable it, restart zcashd with the -experimentalfeatures and\n"
-        "-" + enableArg + " commandline options, or add these two lines\n"
-        "to the zcash.conf file:\n\n"
-        "experimentalfeatures=1\n"
-        + enableArg + "=1\n";
+std::string experimentalDisabledHelpMsg(const std::string& rpc, const std::vector<string>& enableArgs)
+{
+    std::string cmd, config = "";
+    const auto size = enableArgs.size();
+    assert(size > 0);
+
+    auto newline = [](uint16_t number = 1) -> std::string {
+        std::string ret = "";
+        while(number > 0) {
+            ret += "\n";
+            number--;
+        }
+        return ret;
+    };
+
+    for (auto i = 0; i<size; ++i)
+    {
+        if (size == 1 || i == 0)
+        {
+            cmd += "-experimentalfeatures and -" + enableArgs.at(i);
+
+            config += "experimentalfeatures=1" + newline();
+            config += enableArgs.at(i) + "=1" + newline();
+        }
+        else {
+            cmd += " or: " + newline() + "-experimentalfeatures and -" + enableArgs.at(i);
+
+            config += newline() + "or:" + newline(2);
+            config += "experimentalfeatures=1" + newline();
+            config += enableArgs.at(i) + "=1" + newline();
+        }
+    }
+    return "\nWARNING: " + rpc + " is disabled." + newline() +
+        "To enable it, restart zcashd with the following command line options:" + newline()
+        + cmd + newline(2) +
+        "Alternatively add these two lines to the zcash.conf file:" + newline(2)
+        + config;
```

Comment:
```suggestion
    for (size_t i = 0; i < size; ++i)
    {
        if (size == 1 || i == 0)
        {
            cmd += "-experimentalfeatures and -" + enableArgs.at(i);

            config += "experimentalfeatures=1\n";
            config += enableArgs.at(i) + "=1\n";
        }
        else {
            cmd += " or:\n-experimentalfeatures and -" + enableArgs.at(i);

            config += "\nor:\n\n";
            config += "experimentalfeatures=1\n";
            config += enableArgs.at(i) + "=1\n";
        }
    }
    return "\nWARNING: " + rpc + " is disabled.\n" +
        "To enable it, restart zcashd with the following command line options:\n"
        + cmd + "\n\n" +
        "Alternatively add these two lines to the zcash.conf file:\n\n"
        + config;
```

Reply:
Yes, i had it like this before, then added the lambda to clarify a bit the code but actually was not a very good idea. Removed at https://github.com/zcash/zcash/pull/4402/commits/5d3ff01d6177e5fe3974b4483920d794f7e37f12